instrumentation/net/http/otelhttp/example/server/server.go

```diff
@@ -53,6 +58,17 @@ func initTracer() (*sdktrace.TracerProvider, error) {
 	return tp, err
 }
 
+func initMeter() (*sdkmetric.MeterProvider, error) {
+	enc := json.NewEncoder(os.Stdout)
+	exp, err := stdoutmetric.New(stdoutmetric.WithEncoder(enc))
```

Comment:
Why not just use the default tab separated encoder the exporter uses?

```suggestion
	exp, err := stdoutmetric.New()
```